python/pyarrow/_dataset.pyx

```diff
@@ -1174,6 +1189,12 @@ cdef class IpcFileFormat(FileFormat):
 
     def equals(self, IpcFileFormat other):
         return True
+    
+    def make_write_options(self, **kwargs):
+        cdef IpcFileWriteOptions opts = \
+            <IpcFileWriteOptions> FileFormat.make_write_options(self)
+        opts.options = IpcWriteOptions(**kwargs)
```

Comment:
Above, the property is called `write_options`?

Reply:
Thanks, fixed!